workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+Here on the right side in the files section, you can see 3 files that are:
+1. index.html
+2. script.js
+3. style.css.
+
+Now let’s take a look at those and try to understand it individually.
+
+## 1)	The HTML file
+HTML gives the structure to our webpage. Here in the first line, we have `<!DOCTYPE html>` which is used for specifying the version of HTML the document is using. Next, we have the `<title>` tag which specifies the title of our webpage that will be shown on the tab bar. Then we have <body> tag which contains the main visible part of our web page. Make sure that all the code you write is included between the opening and closing tags. ( `<body></body>` )
+  
+  Make sure to remove line 5 (`<meta name="viewport" content="width=device-width">`) which may cause some weird responsiveness issues for our workshop.
+  
+  ![Removing the fifth line](https://cloud-2un6g11k6.vercel.app/2html-1.gif)
+  
+## 2)	The CSS file
+This is the style.css file in your directory. This defines the looks of our website. Whatever styles we give to our website is written in this file. If you take a look in the <head> tag in your HTML file, you will find a line of code (`<link href="style.css" rel="stylesheet" type="text/css" />`). This means that your HTML file is properly linked to your CSS file!
+
+## 3)	The JavaScript file
+This is the script.js file in your directory. It's empty for now, but we will fill it eventually through the workshop! The code written here will be responsible for the functioning of our Wheel. If you take a look in the <body> tag in your HTML file, you will find a line of code (`<script src="script.js"></script>`). This means that your HTML file is properly linked to your JavaScript file!
+
+# Part 4: Building the Wheel
+
+## 1)	HTML
+
+Let’s begin the coding part of our workshop. Here we will write the HTML code required to build our wheel.
+
+Give a division with the help of `<div>` tag inside our body tags. Now give `id="mainbox"` `class="mainbox"` in that division, by this CSS and JavaScript file can communicate for the specific division (`<div id="mainbox" class="mainbox"> </div>` should be the final product for this. 
+
+Create another div with `id="box"` `class="box"`. Nest this inside the mainbox div by putting it in-between `<div ...>` and `</div>`.
+Nesting is like Russian [Matryoshka dolls](https://en.wikipedia.org/wiki/Matryoshka_doll), so take care of it.
+
+Now nest two more divisions in `<div id="box" class="box">` with class as “box1” and “box2” respectively. Just place both of these in between that `<div ...>` and `</div>`.
+Now in the `<div class="box1">` tag, use 4 span tags one by one and give the class as “span1”, “span2”, “span3”, and “span4” respectively.
+
+```html
+<div class="box1">
+  <span class="span1"></span>
+  <span class="span2"></span>
+  <span class="span3"></span>
+  <span class="span4"></span>
+</div>
+```    
+
+Right now, your division should look something like this.
```

Comment:
I'm a little lost here because where are the other divs that you gave instructions to write earlier?